products/sle-micro/main.pm

```diff
@@ -24,6 +27,20 @@ unregister_needle_tags('ENV-OFW-1');
 unregister_needle_tags('ENV-PXEBOOT-0');
 unregister_needle_tags('ENV-PXEBOOT-1');
 
+sub load_boot_from_disk_tests {
+    if (is_s390x()) {
+        loadtest 'installation/bootloader_start';
+        loadtest 'boot/boot_to_desktop';
+    } else {
+        loadtest 'microos/disk_boot';
+    }
+    loadtest 'transactional/host_config';
+    loadtest 'console/suseconnect_scc';
```

Comment:
Should be considered only if SCC_REGISTER is installation?

Reply:
hm....is there a test suite without SCC_REGISTER ? I feel it is kinda implicit when it comes to `slem`

Reply:
yes, for new products without SCC entry yet